### DISCUSSION TITLE
zepetopackage 파일 만드는법

### DISCUSSION THREAD

**Discussion by Park-sechang** [naverz/zepeto-studio-kor#390] [5. Q&A]
말 그대로 zepetopackage 파일 만드는 방법이 알고 싶습니다.

**Comment by minjujuu** [comment]
https://studio.zepeto.me/guides/preparing-for-world-creating
위 링크 하단에 '프로젝트 퍼블리싱'을 참고해보시면 좋을 것 같습니다!

**Reply by Park-sechang** [reply]
감사합니다!

**Comment by peterkangdev** [comment]
프로젝트 폴더/Build/ZEPETO.Script 아래 보면 있습니다.
Publish 버튼을 누른 후 Upload 대화상자가 나오면 만들어져 있을 겁니다.
